src/utilities/codemirror/vml/vmlConstants.ts

```diff
@@ -0,0 +1,95 @@
+// The value here must match the rule and token names in vml.grammar
```

Comment:
We have a `/constants` directory. Does it make sense to put this there?

Reply:
I hesitate to since I think we'll pull this out of core in the medium term

Reply:
Placing it under the `/constants` directory is purely for organizational purposes in order to be easily discoverable.

Reply:
Oh sorry I misunderstood. I thought you wanted to keep it in core for a while. Disregard!